mmv1/third_party/terraform/tests/resource_compute_network_endpoints_test.go.erb

```diff
@@ -0,0 +1,242 @@
+<% autogen_exception -%>
+package google
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+func TestAccComputeNetworkEndpoints_networkEndpointsBasic(t *testing.T) {
+	// Multiple fine-grained resources
+	acctest.SkipIfVcr(t)
```

Comment:
I think it should be possible (and if so, it is preferable) to remove this line, because there will only ever be one of this resource per network endpoint group (at least in the tests) so we shouldn't run into the failure case that would require skipping VCR tests.

(`// Multiple fine-grained resources` should also be deleted because it's the justification.)

(Note that this _is_ a concern for the network_endpoint tests because it has to use multiple resources to accomplish the same thing! :-))

Reply:
Removing this line will also make the tests for this change run automatically on the PR - they are skipped otherwise.

Reply:
Good to know, removed.

Reply:
Seems like the magician is still skipping these tests?

Reply:
I see it as passing in https://github.com/GoogleCloudPlatform/magic-modules/pull/8068#issuecomment-1581892143

Reply:
ah you're right, not sure what I read that as the first time.